features/git-town-set-parent-branch/set_parent_branch.feature

```diff
@@ -8,6 +8,7 @@ Feature: update the parent of a nested feature branch
   Background:
     Given my repository has a feature branch named "parent-feature"
     And my repository has a feature branch named "child-feature" as a child of "parent-feature"
+    And I am on the "child-feature" branch
```

Comment:
The step to create child branches implicitly changed the currently checked out branch. This test was relying on this unintended side effect. This side effect no longer exists, so we make the branch we expect on explicit.